src/components/common/MoveButton.tsx

```diff
@@ -43,7 +43,7 @@ const MoveButton: FC<MoveButtonProps> = ({
   return (
     <GraaspMoveButton
       color={color}
-      type={type}
+      type={type as ActionButtonVariant}
```

Comment:
Is this caused by the nullable? If yes, consider using a non-nullable prop type for `type`, or returning `null` if not set (but I think removing the nullable makes sense).